console/console-init/ui/src/components/MetadataListTable/MetadataListTable.tsx

```diff
@@ -78,10 +79,12 @@ export const MetadataListTable: React.FC<IMetadataListTablePorps> = ({
       {dataList &&
         dataList.map((list: any) => {
           const { headers, data } = list;
+          const metadataOptions = getJsonForMetadata(data);
           return (
             <>
               <TableHeader headers={headers} aria-labelledby={ariaLabelledby} />
-              {data && data.map((row: any) => <TableRow rowData={row} />)}
+              {metadataOptions &&
+                metadataOptions.map((row: any) => <TableRow rowData={row} />)}
```

Comment:
Type missing for row.

Reply:
I tried to use specific type but difficult to use because of row have the value field and value can we anything i..e. string/object/array/array object. many more. Will take care later if possible.